core/privc3/paddle_tensor.h

```diff
@@ -47,6 +49,14 @@ template <typename T> class PaddleTensor : public TensorAdapter<T> {
 
   const T *data() const override { return _tensor.data<T>(); }
 
+  const paddle::framework::Tensor* paddle_tensor() const {
+    return &_tensor;
+  }
+
+  paddle::framework::Tensor* paddle_tensor() {
```

Comment:
rename to mutable_paddle_tensor, so that users can tell it from const paddle_tensor